test/dart_only/debug/dart_cli.test.ts

```diff
@@ -197,17 +197,13 @@ describe("dart cli debugger", () => {
 		ensureVariable(listVariables, "l[1]", "[1]", "1");
 
 		const mapVariables = await getVariables(dc, variables.find((v) => v.name === "m").variablesReference);
-		ensureVariable(mapVariables, `m["l"]`, `0 = ["l"]`, "[2]");
-		ensureVariable(mapVariables, `m["s"]`, `1 = ["s"]`, `"Hello!"`);
-		ensureVariable(mapVariables, undefined, `2 = [DateTime]`, `"today"`);
-		ensureVariable(mapVariables, undefined, `3 = [DateTime]`, `"tomorrow"`);
-		ensureVariable(mapVariables, `m[true]`, `4 = [true]`, `true`);
-		ensureVariable(mapVariables, `m[1]`, `5 = [1]`, `"one"`);
-		ensureVariable(mapVariables, `m[1.1]`, `6 = [1.1]`, `"one-point-one"`);
-
-		const mapListVariables = await getVariables(dc, mapVariables.find((v) => v.name === `0 = ["l"]`).variablesReference);
-		ensureVariable(mapListVariables, `m["l"][0]`, "[0]", "0");
-		ensureVariable(mapListVariables, `m["l"][1]`, "[1]", "1");
+		ensureVariable(mapVariables, undefined, "0", `"l" -> [2]`);
```

Comment:
I think we've lost some useful tests here, that ensure evaluateName is set correctly for the values (the ones that didn't have undefined). This is because they've been pushed down a level now. I think we should add at least some of them back - the `await getVariables` on the lines above similar so you should be able to copy them.